### DISCUSSION TITLE
How can I change the unfocused dim color?

### DISCUSSION THREAD

**Discussion by FrenzyExists** [liuchengxu/vim-clap#735] [Q&A] [answered]
Is there a way to change the dimming of the unfocused part when using clap?
![image](https://user-images.githubusercontent.com/47986470/128819332-cb9b2956-45e7-45e5-a529-d342e4b28c68.png)
Like in this picture, instead of having the near black unfocused color,  I want to either lower the dimming, or change the color of it so it could match with my colorscheme. I checked the avaiable themes, but I couldn't see any option that could state a way to change such. Am I missing something or is simply not possible?

**Comment by romgrk** [comment, marked as answer]
Are you talking about the shadow? If so it's the `ClapShadow` group.

**Reply by FrenzyExists** [reply]
I think it would be the shadow yeah, is there a way to change that?
![image](https://user-images.githubusercontent.com/47986470/128832881-3e81a5e1-4a47-473b-a7da-71cb91baa7e5.png)
I tried that, but no luck

**Reply by romgrk** [reply]
`hi ClapShadow guibg=#000000`

**Reply by FrenzyExists** [reply]
Nice! Ok, so, I did just that, and it seems that for some reason not all of the shadow affects the unfocused area?

![image](https://user-images.githubusercontent.com/47986470/128834265-00356690-de9c-4199-a0f5-a486bd523388.png)

Kinda weird, here's what i put

![image](https://user-images.githubusercontent.com/47986470/128834471-4e26f9d1-b92d-4a06-a6e3-f7837310ca38.png)

only the number columns are affected for some reason

**Reply by FrenzyExists** [reply]
Ok, i think I found the issue, it seems the coloring on my vim theme has some missing  colors, and hence why it looks so weird, so that means that the snippet sent works! :D Lemme mark it answerered, and fix the vim theme itself (which is separate from Clap)
